lib/lib.misc.php

```diff
@@ -43,10 +44,10 @@ function overshoot_time(float $timelimit, string $overshoot_cfg) : float
 function overshoot_parse(float $timelimit, string $token) : float
 {
     $res = sscanf($token, '%d%c%n');
-    if (count($res)!=3) {
+    if (count((array) $res)!=3) {
```

Comment:
why is this cast here necessary?

Reply:
cc @nickygerritsen

Reply:
Beceause in theory sscanf can return false, which cant be used with count(). If we are sure it doesn't, we can add `/** @var array */`, but are we sure? The docs of sscanf are not super clear to me.